.github/workflows/publish-node.yml

```
@@ -10,13 +10,20 @@ on:
+  cancel-in-progress: false
+
 jobs:
   publish:
     runs-on: ubuntu-latest
+    timeout-minutes: 25
     defaults:
       run:
         shell: bash
         working-directory: sdk/node
+    env:
+      CAP_NODE_PACK_DIR: ${{ runner.temp }}/cap-node-package
```

actionlint (1.7.12)

[error] context "runner" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "secrets", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)